spec/mars_rover_kata_spec.rb

```diff
@@ -0,0 +1,37 @@
+require './lib/mars_rover'
+require './lib/commands'
+
+RSpec.describe 'mars rover kata' do
+  describe 'MarsRover' do
+    it 'knows its initial starting coordinates of x and y' do
+      mars_rover = MarsRover.new(x: 200, y: 400, direction: 'S')
+
+      expect(mars_rover.coordinates).to eq [200, 400]
+    end
+
+    it 'knows its compass direction' do
+      mars_rover = MarsRover.new(x: 200, y: 400, direction: 'N')
+
+      expect(mars_rover.direction).to eq 'N'
+    end
+  end
+
+  describe 'Commands' do
+    it 'recieves a character array of commands' do
+      commands = Commands.new
+
+      result = commands.send(['w', 'w', 's'])
```

Comment:
Are 'w', 'w', 's' the direction? The commands are really 'f', 'b', 'l' and 'r' don't forget (c.f. README)